frontend/src/components/teacher/student-management/classroom-summary/ClassroomPopover.tsx

```diff
@@ -112,7 +112,7 @@ const ClassroomPopover = ({
       />
       <Modal
         body={
-          'Are you sure you want to archive this classroom? You can find your archived classrooms in the "Archived" tab.'
+          'Are you sure you want to archive this classroom? You can find your archived classrooms in the "Archived" tab. Note that all upcoming assessments will be deleted and active assessments will end.'
```

Comment:
Wondering about whether you think it would be useful to make this text italic and/or bold and/or in a separate paragraph to draw attention to it?